src/pipx/commands/common.py

```diff
@@ -20,12 +21,36 @@
 def expose_apps_globally(
     local_bin_dir: Path, app_paths: List[Path], package: str, *, force: bool
 ):
-    if WINDOWS:
+    if not _can_symlink(local_bin_dir):
         _copy_package_apps(local_bin_dir, app_paths, package)
     else:
         _symlink_package_apps(local_bin_dir, app_paths, package, force=force)
 
 
+_can_symlink_cache = {}
```

Comment:
would be nice to add mypy types, mostly for documentation so people reading the code can guess how the dict is used

Reply:
Thanks, will do. I forgot you could annotate variables.